classes/Config/Config.php

```diff
@@ -24,7 +24,9 @@ class Config
 {
     const API_VERSION = 'v10.0';
 
-    const REQUIRED_PS_ACCOUNTS_VERSION = '3.2.2';
+    const COMPLIANT_PS_ACCOUNTS_VERSION = '3.0.0';
```

Comment:
Sert à quoi ?

Reply:
For merchants upgrading, I don't want to see the module "broken" until they upgrade PS Account